CHANGELOG.md

```diff
@@ -6,856 +6,858 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [v2.19.1]
+- Uncommented elements needed to build `additive-manufacturing`
+
 ## [v2.19.0] - 2024-06-28
 
-* Bake `exercises` from `note` in `nursing-internal`
-* Remove `BakeMathInParagraph` from `computer-science`
-* Fix locale typo in `lifespan-development`
-* Bake `references` in `lifespan-development`
+- Bake `exercises` from `note` in `nursing-internal`
+- Remove `BakeMathInParagraph` from `computer-science`
+- Fix locale typo in `lifespan-development`
+- Bake `references` in `lifespan-development`
```

Comment:
i notice these were changed - is your editor set to do this automatically? ideally bullet points would stay `*` but if it's hard to avoid this change we can adjust

Reply:
I think it was my Markdown formatter. Will update.